third_party/curl/BUILD.curl.bazel

```diff
@@ -1,14 +1,14 @@
-load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
 
 filegroup(
-    name = "all",
+    name = "all_srcs",
     srcs = glob(["**"]),
 )
 
 _CACHE_ENTRIES = {
     "BUILD_CURL_EXE": "off",
     "BUILD_SHARED_LIBS": "off",
-    "CMAKE_PREFIX_PATH": "$EXT_BUILD_DEPS/openssl",
+    "CMAKE_PREFIX_PATH": ";$EXT_BUILD_DEPS/openssl",
```

Comment:
Why the addition of the `;`?

Reply:
That was an accident. No reason for that to be there

Reply:
sounds good.  you can reassign me to merge once you've had a chance to remove that.

Reply:
Fixed